.pipelines/onebranch/pipeline.buildrp.dev.yml

```diff
@@ -0,0 +1,80 @@
+#################################################################################
+#                               OneBranch Pipelines                             #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
+# Yaml Schema:    https://aka.ms/obpipelines/yaml/schema                        #
+# Retail Tasks:   https://aka.ms/obpipelines/tasks                              #
+# Support:        https://aka.ms/onebranchsup                                   #
+#################################################################################
+
+trigger: none
+pr: none
+
+variables:
+  CDP_DEFINITION_BUILD_COUNT: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
+  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+  DEBIAN_FRONTEND: noninteractive
```

Comment:
Can we use one variable format? We use CamelCase around the place

Reply:
And these are not used. I assume this is somehow related to 
```
extends:
  template: v2/OneBranch.NonOfficial.CrossPlat.yml@templates # https://aka.ms/obpipelines/templates
```
Would be good to understand how.

Reply:
I am not against consistency - these were part of the Onebranch pipeline template and were left untouched.  I can test to see if it breaks anything downstream.

Reply:
From what im reading it should work both ways. All variables can be used either in ENV format or `$(foo)`, depending on how they accesses. But would not be first time Azure devops docs are misleading. 

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#environment-variables

Reply:
The big question was whether or not the Onebranch containers cared.  Tested and it works fine after changing to camel case.